core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -297,6 +302,18 @@ protected void startServing(String startMessage, String stopMessage) {
     LOG.info("Alluxio master ended {}", stopMessage);
   }
 
+  protected void startCommonServices(boolean force, boolean startIfEnabled) {
```

Comment:
How do these two flags work?

Reply:
Add a method java doc comment.

Reply:
This design looks really complicated. Why do you need this?

Reply:
These flags are not yet removed?

Reply:
Refactor it.